### PR TITLE
EN-2006 crop image settings porting from 4.3.5

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/ContentAdminAction.java [hunk 1]
```diff
@@ -48,9 +55,13 @@
     private IResourceManager resourceManager;
     private ICmsSearchEngineManager searchEngineManager;
 
+    private String aspectRatio;
+    private List<String> ratio = new ArrayList<>();
+
     @Override
     public void validate() {
         super.validate();
+        this.validateAspectRatioList();
```
Comment (eugeniosant) [inline plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/ContentAdminAction.java]: The validation doesn't work, and tests for this part are missing

File: plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/ContentAdminAction.java [hunk 2]
```diff
@@ -279,4 +314,25 @@ public void setResourceManager(IResourceManager resourceManager) {
         this.resourceManager = resourceManager;
     }
 
+    @Override
+    protected void initLocalMap() throws Throwable {
+        String xmlParams = this.getConfigParameter();
+        Map<String, String> systemParams = SystemParamsUtils.getParams(xmlParams);
+        this.setSystemParams(systemParams);
+        if (this.getRatio() != null && !this.getRatio().isEmpty()) {
```
Comment (eugeniosant) [inline plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/ContentAdminAction.java]: Add some unit test

**Comment by entando-service** [comment]
All is Well

**Comment by entando-service** [comment]
All is Well

**Comment by entando-service** [comment]
All is Well

**Pull request merged**